pkg/actions/testdata/lua/catalogexport_hive_partition_pager.lua

```diff
@@ -0,0 +1,69 @@
+local hive = require("lakefs/catalogexport/hive")
+
+-- helper function to slice table array
+function table.slice(tbl, first, last, step)
+    local sliced = {}
+
+    for i = first or 1, last or #tbl, step or 1 do
```

Comment:
How can anyone not love Lua?  (No action needed)

Reply:
lol